Examples/test-suite/c_backend_cpp_exception.i

```diff
@@ -0,0 +1,25 @@
+%module c_backend_cpp_exception
+
+%exception {
+  try {
+    $action
+  } catch(SomeKnownException) {
+    checkVal = 1;
+  } catch(...) {
+    checkVal = 2;
+  }
+}
+
+%inline %{
+  class SomeKnownException{};
+  class SomeUnkownException{};
+  int checkVal = 0;
+
+  void throwSomeKnownException(void) {
+    throw SomeKnownException();
+  }
+
+  void throwSomeUnknownException(void) {
+    throw SomeUnkownException();
+  }
+%}
```

Comment:
A number of these new files lack a newline on the final line, which is probably better avoided.

Reply:
AFAICS (or, rather, `rg -l '[^\n]\z'` can see) there are only 4 of such files added on this branch. I can fix them, of course, but there are a bunch of existing files in master with the incomplete last line and I think it might make more sense to fix all of them at once after merging.